examples/motor-imagery/plot_augmented.py

```diff
@@ -72,10 +72,14 @@ def transform(self, X):
 tmin, tmax = 1., 2.
 event_id = dict(hands=2, feet=3)
 subject = 7
-runs = [6, 10, 14]  # motor imagery: hands vs feet
+runs = [6, 10]  # motor imagery: hands vs feet
```

Comment:
Display is broken: without run 14, curves of results are flat.

I suggest to keep run 14, but to reduce grid search:

```
param_grid = {
    'augmenteddataset__order': [2, 3, 4],
    'augmenteddataset__lag': [5, 7, 9],
}
```